src/software/embedded/thunderloop.cpp

```diff
@@ -238,6 +247,10 @@ void Thunderloop::runLoop()
                 // Save new primitive
                 primitive_ = new_primitive;
 
+                auto orientation_msg = primitive_.orientation();
```

Comment:
avoid `auto` unless it's a really long type name. It makes the code harder to read for newer members (especially since we have multiple "angle"-ish types in the codebase)